x-pack/plugin/core/src/main/java/module-info.java

```diff
@@ -228,6 +228,7 @@
     exports org.elasticsearch.xpack.core.watcher.trigger;
     exports org.elasticsearch.xpack.core.watcher.watch;
     exports org.elasticsearch.xpack.core.watcher;
+    exports org.elasticsearch.xpack.cluster.settings;
```

Comment:
That package is where I added the `ClusterSettings` class where the shared `CLUSTER_LOGSDB_ENABLED` setting is available. This is required because now that setting is used by `x-pack:stack` and `x-pack:logsdb`...so I need it to be in a common place and the package being exported.